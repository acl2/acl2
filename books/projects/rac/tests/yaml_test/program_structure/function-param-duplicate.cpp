// RAC begin

int foo(int a, int A) { return 0; }
