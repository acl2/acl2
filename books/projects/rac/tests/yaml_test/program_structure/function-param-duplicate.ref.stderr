function-param-duplicate.cpp:3 (19-20): A is already used but with a different case (in ACL2, they will refer to the same thing)
  3 | int foo(int a, int A) { return 0; }
    |                    ^
note: first defined here
  3 | int foo(int a, int A) { return 0; }
    |             ^
